tests/issues.ts

```diff
@@ -101,3 +102,33 @@ describe('issues/12', () => {
     assert.deepEqual(patch_results, [null, null, null])
   })
 })
+
+describe('issues/29', () => {
+  var input = {
```

Comment:
Ok, I thought of an edge-case!

Replace these three vars with:
```js
  var input = {
    root: {
      diffed: ['a', 'b'],
      not_diffed: ['a', 'b'],
    },
  }
  var output = {
    root: {
      diffed: ['a'],
      not_diffed: ['a'],
    }
  }
  var expected_patch = [
    {op: 'remove', path: '/root/diffed/1'},
    {op: 'replace', path: '/root/not_diffed', value: ['a']},
  ]
```

This fails, saying:
```patch
+ expected 
- actual

     "op": "remove"
     "path": "/root/diffed/1"
   }
   {
-    "op": "remove"
-    "path": "/root/not_diffed/1"
+    "op": "replace"
+    "path": "/root/not_diffed"
+    "value": [
+      "a"
+    ]
   }
 ]
```